native/src/modules/layout/containers/HeaderContainer.js

```diff
@@ -87,13 +107,21 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
       }
     })
   }
+
   const peeking = isPeekingRoute(state, { routeCity: route.city })
-  const shareUrlFromScene: ?string = typeof ownProps.scene.route.params?.shareUrl === 'string' ? ownProps.scene.route.params?.shareUrl : null
-  const { city, language } = route
-  const path = route.path || null
-  const shareUrl = shareUrlFromScene || getShareUrl({ city, language, path, routeName: ownProps.scene.route.name })
+  const path = route.path || undefined
+  const language = route.language
+  let shareUrl = url(path) // getShareUrl({ city, language, path, routeName: ownProps.scene.route.name })
+
+  if (ownProps.scene.route.name === NEWS_ROUTE && route?.type && stateCityCode) {
+    shareUrl = cityContentUrl({ cityCode: stateCityCode, languageCode: language, route: NEWS_ROUTE, path: route.type })
```

Comment:
So sharing a single news is still not possible, right? Could you fix it or create an issue please?

Reply:
created an issue